### PR TITLE
Make rust-lldb warn about unsupported versions of LLDB

### PR DESCRIPTION
**Pull request by michaelwoerister**
r? @alexcrichton

### CODE REVIEW

File: src/etc/rust-lldb [hunk 1]
```diff
@@ -12,6 +12,17 @@
 # Exit if anything fails
 set -e
 
+LLDB_VERSION=`lldb --version 2>/dev/null | head -1 | cut -d. -f1`
+
+if [ "$LLDB_VERSION" = "lldb-350" ]
+then
+    echo "***"
+	echo \
+"WARNING: This version of LLDB has known issues with Rust and cannot not \
```
Comment (aravind-pg) [inline src/etc/rust-lldb]: "cannot not"?

Comment (michaelwoerister) [inline src/etc/rust-lldb]: fixed, thanks!

**Pull request merged**